dpdd.tex

```diff
@@ -193,14 +192,16 @@ \subsection{Classes of LSST Data Products}
+\item \newtext{\textbf{Data Release Processing}, which is performed on an annual cadence\footnote{Except for the first two data releases, which will be created six months apart.} and produces \textbf{Data Release} data products. These processing campaigns use all data collected by the survey to-date in order to perform a comprehensive analysis, including measurement of both faint static objects through the construction of deep coadds, and light curve characterization through difference imaging across all available epochs.}
+    The data products produced will include the calibrated single-epoch images, deep coadds, catalogs of characterized \Objects (detected on deep coadds as well as individual visits\footnote{The LSST takes two exposures per pointing, nominally 15 seconds in duration each, called \emph{snaps}. For the purpose of data processing, that pair of exposures will typically be coadded and treated as a single exposure, called a \emph{visit}.}), \Sources\footnote{When written in bold monospace type (i.e., \texttt{\textbackslash{}tt}), \Objects and \Sources refer to objects and sources detected and measured as a part of Data Release processing.} (detections and measurements on individual visits), and \ForcedSources (constrained measurement of flux on individual visits). \newtext{It will also include difference imaging tables similar to those produced during Prompt Processing} (see \S \ref{sec:l1dbreproc}). In contrast to the \oldtext{``living''} \PPDB, which is updated \oldtext{in real-time} \newtext{continuously during observing}, the \DR is static and will not change after release. Data Release data products are discussed in \S~\ref{sec:level2}.
```

Comment:
(If you take my suggestion above, this shouldn't refer to prompt products as “difference imaging tables”.)